.github/ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,25 @@
-Describe in one or two sentences why you're opening this issue:
+### Description:
+<!-- Describe in one or two sentences why you're opening this issue: -->
+> **REPLACE-ME**
 
+### Tell us a bit about yourself:
+* Version of JekyllAdmin I'm using <HINT: use `bundle show` to check>: ***REPLACE-ME***
```

Comment:
Why add the `**REPLACE ME**` text? I feel like that's creating more work for a bug reporter (highlight and delete 5 times), than just having a colon and a blank which should achieve the same effect.